deployment/7_deploy_2.5.js

```diff
@@ -76,6 +77,8 @@ const main = async () => {
   const DappRegistryWrapper = await DappRegistry.new(config.settings.timelockPeriod);
```

Comment:
Shouldn't we create the DappRegistry with a timelock of 0 and add the first set of dapps in the script? Then chenage the timelock to its correct value and set the multisig as the owner?

Reply:
Sure, #208 already does that so when that's merged I'll rebase to update here.